up42/catalog.py

```diff
@@ -283,3 +285,114 @@ def download_quicklooks(
+        """
+        if not isinstance(scene, Series):
+            raise ValueError(
+                "`scene` parameter must be a GeoSeries, or a single item/row of a GeoDataFrame. "
+                "For instance, search_results.loc[0] returns a GeoSeries."
+            )
+        aoi_fc = any_vector_to_fc(
+            vector=geometry,
+        )
+        aoi_geometry = fc_to_query_geometry(
+            fc=aoi_fc,
+            geometry_operation="intersects",
+            squash_multiple_features="union",
+        )
+        data_provider_name = scene.providerName
+        order_params = {"id": scene.id, "aoi": aoi_geometry}
+        return data_provider_name, order_params
+
+    def estimate_order(
+        self,
+        geometry: Union[
+            Dict,
+            Feature,
+            FeatureCollection,
+            List,
+            GeoDataFrame,
+            Point,
+            Polygon,
+        ],
+        scene: Series,
+    ) -> int:
+        """
+        Estimate the cost of an order from an item/row in a result of `Catalog.search`.
+
+        Args:
+            geometry (Union[ Dict, Feature, FeatureCollection, List, GeoDataFrame, Point, Polygon, ]): The intended
+                output AOI of the order.
+            scene (GeoSeries): A single item/row of the result of `Catalog.search`. For instance, search_results.loc[0]
+                for the first scene of a catalog search result.
+
+        Returns:
+            int: An estimated cost for the order in UP42 credits.
+        """
+        data_provider_name, order_params = self._order_payload(geometry, scene)
+        return Order.estimate(self.auth, data_provider_name, order_params)
+
+    def place_order(
+        self,
+        geometry: Union[
+            Dict,
+            Feature,
+            FeatureCollection,
+            List,
+            GeoDataFrame,
+            Point,
+            Polygon,
+        ],
+        scene: Series,
+        track_status: bool = False,
+        report_time: int = 120,
+    ) -> "Order":
+        """
+        Place an order from an item/row in a result of `Catalog.search`.
+
+        Args:
+            geometry (Union[ Dict, Feature, FeatureCollection, List, GeoDataFrame, Point, Polygon, ]): The intended
+                output AOI of the order.
+            scene (GeoSeries): A single item/row of the result of `Catalog.search`. For instance, search_results.loc[0]
+                for the first scene of a catalog search result.
+            track_status (bool): If set to True, will only return the Order once it is `FULFILLED` or `FAILED`.
+            report_time: The intervall (in seconds) when to get the order status, if `track_status` is set tp True.
+
+         Warning:
+            When placing orders of items that are in archive or cold storage,
+            the order fulfillment can happen up to **24h after order placement**.
+            In such cases,
+            please make sure to set an appropriate `report_time`.
```

Comment:
Add "Or track the status later on via `order.track_status()`" ?

Reply:
Well I would say if you're ordering from archive you probably won't use track_status at all no? You'd just order and later on check the status? What do you think?

Reply:
Yeah, I imagined it like that. Someone firing off the order, and checking 24h later via code if it is complete. Optional, but could give here a hint or code reference linking how to do that.

Reply:
Added